pom.xml

```diff
@@ -33,22 +33,28 @@
   <dependencyManagement>
     <dependencies>
       <dependency>
-        <groupId>org.wildfly.swarm</groupId>
+        <groupId>io.thorntail</groupId>
         <artifactId>bom</artifactId>
-        <version>${version.wildfly.swarm}</version>
+        <version>${version.thorntail}</version>
         <type>pom</type>
         <scope>import</scope>
       </dependency>
+      <!-- workaround for https://issues.jboss.org/browse/THORN-2015 for running Arquillian tests -->
+      <dependency>
```

Comment:
Why not use io.thorntail:config-api-runtime:1.4.0 ?

Reply:
Because that leaves `org.wildfly.swarm:config-api-runtime` present. Excluding `org.wildfly.swarm:config-api-runtime` is enough for uberjar mode, but in classpath mode, the exclusion is ignored and `org.wildfly.swarm:config-api-runtime` is brought in anyway. It then depends on what's "earlier" in a `HashSet`, in an iteration order, and `org.wildfly.swarm:config-api-runtime` often wins. An old version of it, actually. Managing `org.wildfly.swarm:config-api-runtime` to 1.3.1 does work fine, because there's no difference between `org.wildfly.swarm:config-api-runtime:1.3.1` and `io.thorntail:config-api-runtime:1.4.0` from code perspective.